src/kepler_model/train/profiler/node_type_index.py

```diff
@@ -77,7 +77,7 @@ def discover_spec_values():
         vendor = format_vendor(cpu_info["vendor_id_raw"])
 
     cores = psutil.cpu_count(logical=True)
-    chips = max(1, int(subprocess.check_output('cat /proc/cpuinfo | grep "physical id" | sort -u | wc -l', shell=True)))
+    chips = max(1, int(subprocess.check_output('grep --count "physical id" /proc/cpuinfo', shell=True)))
```

Comment:
@sunya-ch  on a multi-cpu system do we not want to count all `physical id` s ?

Reply:
Reverting to just remove `cat`

Reply:
We do. max 1 is to avoid zero.